helm/hpcc/templates/_helpers.tpl

```diff
@@ -315,41 +315,27 @@ Pass in root and includeLabels (optional)
 {{- $dataStorage := ($storage.dataStorage | default dict) -}}
 {{- $planes := ($storage.planes | default list) -}}
 {{- $includeLabels := .includeLabels | default list -}}
-{{- $filterName := .name | default "" -}}
+{{- $includeNames := .includeNames | default list -}}
 {{- range $plane := $planes -}}
  {{- if or ($plane.pvc) (hasKey $plane "storageClass") -}}
   {{- $matchedLabels := include "hpcc.doesStorageLabelsMatch" (dict "plane" $plane "includeLabels" $includeLabels) -}}
-  {{- if ne $matchedLabels "" }}
-   {{- if or (not $filterName) (eq $filterName $plane.name) }}
-    {{- $pvc := hasKey $plane "pvc" | ternary $plane.pvc (printf "%s-%s-pvc" (include "hpcc.fullname" $) $plane.name) -}}
-    {{- $num := int ( $plane.numDevices | default 1 ) -}}
-    {{- if le $num 1 }}
+  {{- if or (ne $matchedLabels "") (has $plane.name $includeNames) }}
```

Comment:
isn't if (ne $matchedLabels ""), the same as if ($matchedLabels) ?

Reply:
I don't think $matchedLabels will work in an or - it will expect a boolean.

Reply:
I've just checked, and it does work...  I will change.